autogen/runtime_logging.py

```diff
@@ -39,6 +41,7 @@ def log_chat_completion(
     invocation_id: uuid.UUID,
     client_id: int,
     wrapper_id: int,
+    source: Union[str, Agent],
```

Comment:
instead of calling it `source`, can we just call it `agent` to make it explicit.